README.md

```diff
@@ -24,237 +104,4 @@ import CustomCloudConnector from '@cesarbr/knot-fog-connector-customcloud';
 
 ...
 const connector = new CustomCloudConnector(config);
-await connector.start();
```

Comment:
Why are you removing the API specification?

Reply:
+1

Reply:
Oops! :eyes: That was by accident! Fixed this :smile: